ros_ign_bridge/test/test_utils.hpp

```diff
@@ -0,0 +1,1564 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
It looks like you removed the `.h` file and added the `.hpp` as a new one. When renaming / moving files, we should always use `git mv` to keep their history. Now it's hard to tell what was already there and what you changed.

Reply:
https://github.com/ignitionrobotics/ros_ign/pull/98/commits/9da33251439227f19e31e5a2ca6a1ed84a60ac9c